msal/src/main/java/com/microsoft/identity/client/internal/controllers/BrokerActivity.java

```diff
@@ -0,0 +1,95 @@
+//  Copyright (c) Microsoft Corporation.
+//  All rights reserved.
+//
+//  This code is licensed under the MIT License.
+//
+//  Permission is hereby granted, free of charge, to any person obtaining a copy
+//  of this software and associated documentation files(the "Software"), to deal
+//  in the Software without restriction, including without limitation the rights
+//  to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions :
+//
+//  The above copyright notice and this permission notice shall be included in
+//  all copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+package com.microsoft.identity.client.internal.controllers;
+
+import android.app.Activity;
+import android.content.Intent;
+import android.os.Bundle;
+
+public final class BrokerActivity extends Activity {
+
+    public static final String BROKER_INTENT = "broker_intent";
+    static final String BROKER_INTENT_STARTED = "broker_intent_started";
+    static final int BROKER_INTENT_REQUEST_CODE = 1001;
```

Comment:
I recall @danieldobalian asking whether or not this value could change between MSAL/ADAL. Possibly we address that?

Reply:
this was actually not for broker....